### PR TITLE
chore(README.md): reflect recent architecture changes

### PR DESCRIPTION
**Pull request by mary-grace**
Discussed with @MaxTru. Will need additional updates with continued architecture changes throughout 8.6 alphas.

### CODE REVIEW

**Comment by mary-grace** [comment]
@felix-mueller Could you take a look at this to validate the changes and push it live? Thanks!

Tagging in @MaxTru as well in case you have further thoughts on how you want this to read.

File: README.md [hunk 1]
```diff
@@ -1,10 +1,19 @@
-# Zeebe - Workflow Engine for Microservices Orchestration
+# Camunda 8 orchestrates complex business processes that span people, systems, and devices
 
 [![Maven Central](https://maven-badges.herokuapp.com/maven-central/io.camunda.zeebe/camunda-zeebe/badge.svg)](https://maven-badges.herokuapp.com/maven-central/io.camunda.zeebe/camunda-zeebe)
 
-Zeebe provides visibility into and control over business processes that span multiple microservices. It is the engine that powers [Camunda Platform 8](https://camunda.com/platform/zeebe/).
+Camunda 8 delivers scalable, on-demand process automation as a service. Camunda 8 is combined with powerful execution engines for BPMN processes and DMN decisions, and paired with tools for collaborative modeling, operations, and analytics.
 
-**Why Zeebe?**
+Camunda 8 consists of six components:
```
Comment (MaxTru) [inline README.md:7]:
```suggestion
Camunda 8 consists of several components:
```

File: README.md [hunk 2]
```diff
@@ -1,10 +1,19 @@
...
+
+* [Console](https://docs.camunda.io/docs/components/console/introduction-to-console/) - Configure and deploy clusters with Console.
+* [Web Modeler](https://docs.camunda.io/docs/components/modeler/about-modeler/) - Collaborate, model processes, and deploy or start new instances.
+* [Zeebe](https://docs.camunda.io/docs/components/zeebe/zeebe-overview/) - The cloud-native process engine of Camunda 8.
+* [Tasklist](https://docs.camunda.io/docs/components/tasklist/introduction-to-tasklist/) - Complete tasks which require human input.
+* [Operate](https://docs.camunda.io/docs/components/operate/operate-introduction/) - Manage, monitor, and troubleshoot your processes.
+* [Optimize](https://docs.camunda.io/optimize/components/what-is-optimize/) - Improve your processes by identifying constraints in your system.
+
```
Comment (MaxTru) [inline README.md:15]:
```suggestion
* [Console](https://docs.camunda.io/docs/components/console/introduction-to-console/) - Configure and deploy clusters with Console.
* Modelers 
  * [Web Modeler](https://docs.camunda.io/docs/components/modeler/about-modeler/) - Web Application to model BPMN, DMN, & Forms and deploy or start new instances.
  * [Desktop Modeler](https://docs.camunda.io/docs/next/components/modeler/desktop-modeler/) - Use Desktop Modeler as desktop application for modeling BPMN, DMN, and Forms with your local process application project.
* [Zeebe](https://docs.camunda.io/docs/components/zeebe/zeebe-overview/) - The cloud-native process engine of Camunda 8.
* [Tasklist](https://docs.camunda.io/docs/components/tasklist/introduction-to-tasklist/) - Complete tasks which require human input.
* [Operate](https://docs.camunda.io/docs/components/operate/operate-introduction/) - Manage, monitor, and troubleshoot your processes.
* [Optimize](https://docs.camunda.io/optimize/components/what-is-optimize/) - Improve your processes by identifying constraints in your system.
* [Connectors](https://docs.camunda.io/docs/next/components/connectors/introduction-to-connectors/) - Integrate with an external system by using a Connector.

```

**Review by MaxTru — CHANGES_REQUESTED** [review]

**Review by mary-grace — COMMENTED** [review]
Thanks for the edits, @MaxTru . Any further thoughts before this goes live?

File: README.md [hunk 3]
```diff
@@ -1,10 +1,22 @@
...
+* Modelers 
+  * [Web Modeler](https://docs.camunda.io/docs/components/modeler/about-modeler/) - Web Application to model BPMN, DMN, & Forms and deploy or start new instances.
+  * [Desktop Modeler](https://docs.camunda.io/docs/next/components/modeler/desktop-modeler/) - Use Desktop Modeler as desktop application for modeling BPMN, DMN, and Forms with your local process application project.
+* [Zeebe](https://docs.camunda.io/docs/components/zeebe/zeebe-overview/) - The cloud-native process engine of Camunda 8.
+* [Tasklist](https://docs.camunda.io/docs/components/tasklist/introduction-to-tasklist/) - Complete tasks which require human input.
+* [Operate](https://docs.camunda.io/docs/components/operate/operate-introduction/) - Manage, monitor, and troubleshoot your processes.
+* [Optimize](https://docs.camunda.io/optimize/components/what-is-optimize/) - Improve your processes by identifying constraints in your system.
+* [Connectors](https://docs.camunda.io/docs/next/components/connectors/introduction-to-connectors/) - Integrate with an external system by using a Connector.
+
```
Comment (MaxTru) [inline README.md:18]:
```suggestion
This repository contains the core execution cluster components of Camunda 8 being:

* [Zeebe](https://docs.camunda.io/docs/components/zeebe/zeebe-overview/) - The cloud-native process engine of Camunda 8.
* [Tasklist](https://docs.camunda.io/docs/components/tasklist/introduction-to-tasklist/) - Complete tasks which require human input.
* [Operate](https://docs.camunda.io/docs/components/operate/operate-introduction/) - Manage, monitor, and troubleshoot your processes.
* [Optimize](https://docs.camunda.io/optimize/components/what-is-optimize/) - Improve your processes by identifying constraints in your system.

Next to the core execution cluster components, the Camunda 8 stack also includes:

* [Console](https://docs.camunda.io/docs/components/console/introduction-to-console/) - Configure and deploy clusters with Console.
* Modelers 
  * [Web Modeler](https://docs.camunda.io/docs/components/modeler/about-modeler/) - Web Application to model BPMN, DMN, & Forms and deploy or start new instances.
  * [Desktop Modeler](https://docs.camunda.io/docs/next/components/modeler/desktop-modeler/) - Use Desktop Modeler as desktop application for modeling BPMN, DMN, and Forms with your local process application project.
* [Connectors](https://docs.camunda.io/docs/next/components/connectors/introduction-to-connectors/) - Integrate with an external system by using a Connector.

```

**Review by MaxTru — CHANGES_REQUESTED** [review]
One suggestion to be more specific about what is in the repo and what is not in it. WDYT?

File: README.md (hunk 3 above)
Comment (mary-grace) [reply README.md:18]: @MaxTru Would you then suggest linking to the repos for these components that aren't in the monorepo? Or continue to link to the documentation?

Comment (MaxTru) [reply README.md:18]: I would continue to link to docs (as-is)

**Comment by mary-grace** [comment]
@MaxTru Please give this one final review and push live if you are in agreement with the changes.

**Review by MaxTru — APPROVED** [review]

**Pull request merged**